en/includes/guides/customize/actions/understanding-actions.md

```diff
@@ -175,6 +173,9 @@ If the error response includes an acceptable payload, it is treated as an error
 </tbody>
 </table>
 
+!!! note
+    Refer to [Configuring HTTP Client Connections]({{base_path}}/guides/customize/actions/setting-up-actions/#configuring-http-client-connections) to adjust timeouts, connection pooling, and retries in deployment.toml for optimal performance.
```

Comment:
Let's update the reference here accordingly

Reply:
Updated in